pkg/testdata/config.yaml

```diff
@@ -3,6 +3,15 @@ inventory: datacenters/hetzner/inventory
 base_dir: config
 org_name: chaordic
 
+terraform_state:
+  backend: s3
+  config:
+    endpoint: endpoint_to_s3_compatible_storage
+    bucket: bucket_name
+    region: auto
+    access_key: your_access_key
+    secret_key: your_secret_key
```

Comment:
Can we do the same trick I did with the Ansible variables for  the keys here?
Check if the "value" exists as an environment value, use that if it exists? (thus not needing to commit any secrets into git).

Reply:
Added in the latest commit